bad_connector_woocommerce/models/woo_backend/common.py

```diff
@@ -495,6 +495,7 @@ def export_sale_order_status(self):
                     ("woo_bind_ids.backend_id", "=", backend.id),
                     ("is_final_status", "!=", True),
                     ("picking_ids.state", "=", "done"),
```

Comment:
Remove this condition because DO state can be done or cancel and we already added the condition of has_done_picking.